proxy/Dockerfile

```
@@ -0,0 +1,33 @@
+WORKDIR /usr/src/app
+
+# Copy the entire workspace context to the container
+COPY . .
+
+# Build only the `proxy` package
+RUN cargo build --release --package proxy
+
+FROM ubuntu:22.04
+
+# Install OpenSSL and other necessary libraries
+RUN apt-get update && apt-get install -y \
```

Hadolint

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)